storage/gcloud/aio/storage/storage.py

```diff
@@ -172,6 +173,22 @@ async def _headers(self) -> Dict[str, str]:
             'Authorization': f'Bearer {token}',
         }
 
+    async def get_buckets(
+        self, project: str, timeout: int = DEFAULT_TIMEOUT,
```

Comment:
Looks like there are [several extra optional params here](https://cloud.google.com/storage/docs/json_api/v1/buckets/list#parameters), it'd be awesome to get those included.